src/Tools/Console/Command/CommandCompatibility.php

```diff
@@ -9,10 +9,32 @@
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
-if ((new ReflectionMethod(Command::class, 'execute'))->hasReturnType()) {
+// Symfony 8
```

Comment:
Per `composer.json`, Symfony 8 isn't supported: https://github.com/doctrine/dbal/blob/b37d160498ea91a2382a2ebe825c4ea6254fc0ec/composer.json#L50

Should it be updated?

Reply:
Not yet, Symfony 8 is still a moving target until November.